modules/tinymce/src/core/test/ts/browser/keyboard/HomeEndKeysTest.ts

```diff
@@ -95,5 +114,25 @@ describe('browser.tinymce.core.keyboard.HomeEndKeysTest', () => {
       TinyContentActions.keystroke(editor, Keys.end());
       TinyAssertions.assertSelection(editor, [ 0, 0 ], 1, [ 0, 0 ], 1);
     });
+
+    context('Inline element', () => {
+      it('move caret at the end on the line', () => {
+        const editor = hook.editor();
+        editor.setContent('<p>test <a href="google.com">link</a> test</p>');
+        TinySelections.setCursor(editor, [ 0, 1, 0 ], 0);
+        TinySelections.setSelection(editor, [ 0, 1, 0 ], 0, [ 0, 1, 0 ], 5);
+        TinyContentActions.keystroke(editor, Keys.end());
+        TinyAssertions.assertCursor(editor, [ 0, 2 ], 5);
+      });
+
+      it('move caret at the beginning on the line (parent) if the first element is an inline element', () => {
```

Comment:
Same deal here just add tests for the inline boundaries element override.

Reply:
updated